src/core/Akka/Akka.csproj

```diff
@@ -11,15 +11,19 @@
     <EmbeddedResource Include="Configuration\akka.conf" />
   </ItemGroup>
 
+  <!-- Common packages for all target frameworks -->
   <ItemGroup>
-      <PackageReference Include="Polyfill" Version="1.28.0" PrivateAssets="all" />
-      <PackageReference Include="Microsoft.Extensions.ObjectPool" Version="$(MsExtVersion)" />
-      <PackageReference Include="Newtonsoft.Json" Version="$(NewtonsoftJsonVersion)" />
-      <PackageReference Include="System.Reflection.Emit" Version="4.7.0" />
-      <PackageReference Include="System.Collections.Immutable" Version="$(MicrosoftLibVersion)" />
-      <PackageReference Include="System.Threading.Channels" Version="$(MicrosoftLibVersion)" />
-      <PackageReference Include="System.Configuration.ConfigurationManager" Version="$(ConfigurationManagerVersion)"/>
-      
+    <PackageReference Include="Microsoft.Extensions.ObjectPool" Version="$(MsExtVersion)" />
+    <PackageReference Include="Newtonsoft.Json" Version="$(NewtonsoftJsonVersion)" />
+    <PackageReference Include="System.Collections.Immutable" Version="$(MicrosoftLibVersion)" />
+    <PackageReference Include="System.Configuration.ConfigurationManager" Version="$(ConfigurationManagerVersion)"/>
+  </ItemGroup>
+
+  <!-- Packages only for NetStandard -->
+  <ItemGroup Condition="'$(TargetFramework)' == '$(NetStandardLibVersion)'">
```

Comment:
- Don't need Polyfill on newer versions of .NET where `record` et al are natively supported
- Don't need System.Reflection.Emit (built into BCL)
- Don't need System.Threading.Channels (built into BCL)

These are pre-requisites for supporting https://github.com/akkadotnet/akka.net/issues/6855 - which is going to require a System.Diagnostics.Activity dependency for `netstandard2.0` only.